### PR TITLE
ux around loading, increased batch size

### PR DESCRIPTION
**Pull request by jwsharpe**
* moved visibility as  a <thead>
* increased batch size to 100
* moved buffer loading to when visibility reaches 50%

### CODE REVIEW

File: app/components/FileLoader.js [hunk 1]
```diff
@@ -320,25 +321,29 @@ export default class FileLoader extends Component {
...
-    const bufferMoreFiles = () => {
+    const bufferMoreFiles = (e, { calculations }) => {
       const start = filesOffset;
       const end = Math.min(start + GAME_BATCH_SIZE, allFiles.length);
+      console.log(calculations.percentagePassed);
 
-      if (start < allFiles.length)
-      {
+      if (this.state.firstLoad) {
+        const nextFilesToRender = allFiles.slice(start, end);
```
Comment (boulajp) [inline app/components/FileLoader.js:330]: Suggestion: You shouldn't need both the if and the else - when its the first load, we'll hit the `if` block and set `firstLoad` to false. We should be able to just do something like this:

`
      if (
        start < allFiles.length &&
        calculations.percentagePassed > 0.5
      ) {
        const nextFilesToRender = allFiles.slice(start, end);
        this.setState({
          filesToRender: filesToRender.concat(nextFilesToRender),
          filesOffset: end,
          firstLoad: false
        });
`

File: app/components/FileLoader.js [hunk 2]
```diff
@@ -320,25 +321,29 @@ export default class FileLoader extends Component {
       this
     );
 
-    const bufferMoreFiles = () => {
+    const bufferMoreFiles = (e, { calculations }) => {
       const start = filesOffset;
       const end = Math.min(start + GAME_BATCH_SIZE, allFiles.length);
+      console.log(calculations.percentagePassed);
```
Comment (boulajp) [inline app/components/FileLoader.js:327]: Fix: We probably shouldn't randomly log this

**Comment by boulajp** [comment]
Thanks for this @jwsharpe! I didn't realize that there was a `percentagePassed` that we could take advantage of to pre-load rows.

File: app/components/FileLoader.js (hunk 2 above)
Comment (jwsharpe) [reply app/components/FileLoader.js]: went and removed it.

**Comment by jwsharpe** [comment]
Sorry about all the prettier diffs

File: app/components/FileLoader.js (hunk 1 above)
Comment (jwsharpe) [reply app/components/FileLoader.js]: needed the firstload just for the issues with percentagePassed on an empty frame. 
However there might be a better variable name.

**Pull request merged**